website/notifications/exceptions.py

```diff
@@ -0,0 +1,8 @@
+from website.exceptions import NodeError
+
+class InvalidSubscriptionError(NodeError):
```

Comment:
I don't think this needs to subclass `NodeError`. It isn't specific to nodes (it can be raised for user subscriptions). Also, `message_short` and `message_long` aren't useful, since the error is never reraised as an HTTPError.